tensorpipe/test/channel/channel_test.cc

```diff
@@ -54,15 +54,15 @@ TEST_P(ChannelTest, ClientToServer) {
         ctx->join();
       },
       [&](std::shared_ptr<transport::Connection> conn) {
-        std::shared_ptr<Context> ctx = GetParam()->makeContext("client");
+        std::shared_ptr<CpuContext> ctx = GetParam()->makeContext("client");
         auto channel = ctx->createChannel(std::move(conn), Endpoint::kConnect);
 
         std::vector<uint8_t> data(dataSize);
 
         // Perform recv and wait for completion.
         auto descriptor = peers_->recv(PeerGroup::kClient);
-        std::future<Error> recvFuture =
-            recvWithFuture(channel, descriptor, data.data(), data.size());
+        std::future<Error> recvFuture = recvWithFuture(
+            channel, descriptor, CpuTensor{data.data(), dataSize});
```

Comment:
Any reason to replace `data.size()` with `dataSize`? I see you're changing it in some tests but not others, which creates inconsistency...

Reply:
I meant to homogeneize the whole thing, but will revert to using `data.size()`.